docs/helpers/FileSystem.md

```diff
@@ -1,6 +1,6 @@
 ---
-permalink: helpers/FileSystem
-editLink: https://github.com/Codeception/CodeceptJS/blob/master/lib/helper/FileSystem.js
```

Comment:
Is there any reason to remove these links?

Reply:
This was not working in Vuepress :(